contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/util/SimpleHttp.java

```diff
@@ -870,6 +928,22 @@ public static HttpStoragePlugin getStoragePlugin(DrillbitContext context, String
     }
   }
 
+  public static String addBackTicksToAliasName(String plugin) {
```

Comment:
The identifier quoting character in Drill is configurable and not necessarily a backtick. See  https://drill.apache.org/docs/lexical-structure/#identifier-quotes. Is there any existing code for unwrapping and wrapping identifiers like this that could be reused?

Reply:
I fixed this to use the planner option.